examples/flutter_gallery/lib/demo/customized/stats/full_stats.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+const int _kAnimationDuration = 2100;
+
+const TextStyle _figureStyle = TextStyle(
+  fontSize: 24.0,
+  fontWeight: FontWeight.bold,
+  color: Colors.black,
+);
+const TextStyle _titleStyle = TextStyle(
+  fontSize: 9.0,
+  fontWeight: FontWeight.w400,
+  color: Colors.black,
+);
+
+class FullStats extends StatefulWidget {
+
+  const FullStats({ Key key }) : super(key: key);
+
+  @override
+  FullStatsState createState() => FullStatsState();
+}
+
+class FullStatsState extends State<FullStats> with TickerProviderStateMixin {
+
+  AnimationController _animationController;
+  Animation<double> _numberCounterAnimation;
+
+  @override
+  void initState() {
+    super.initState();
+    _configureAnimations();
+    animate();
```

Comment:
Doing this both implicitly and explicitly might make it hard to keep track of. Is this needed?

Reply:
Good catch. Wasn't needed.